velox/vector/tests/VectorTest.cpp

```diff
@@ -849,6 +856,11 @@ TEST_F(VectorTest, createOther) {
   testFlat<TypeKind::DATE>(DATE(), vectorSize_);
 }
 
+TEST_F(VectorTest, createDecimal) {
+  //  testFlat<TypeKind::SHORT_DECIMAL>(SHORT_DECIMAL(10, 5), vectorSize_);
```

Comment:
should this be enabled?

Reply:
Yes. The LONG_DECIMAL tests were failing. Disabled it for debugging. Will re-enabled.